packages/asset-service/src/service/AssetService.ts

```diff
@@ -47,6 +47,11 @@ type ByTokenIdArgs = {
   tokenId?: string
 }
 
+type DescriptionData = {
+  description: string
+  isTrusted: boolean
```

Comment:
seems like it might be convenient to be able to omit this field in literals; that said, if so, allowing both `false` and `undefined` would be denormalized.

```suggestion
  isTrusted?: true
```

Reply:
i like this too

Reply:
Done